model/team.go

```diff
@@ -58,6 +58,10 @@ type TeamForExport struct {
 	SchemeName *string
 }
 
+type TeamId struct {
```

Comment:
This feels too generic and odd to be placed here. I would recommend having something like `CommandMoveRequest` kinda like `CommandResponse` that accepts necessary fields, or use a map, kinda like https://github.com/mattermost/mattermost-server/blob/6e6a44764cafdfbe71baf8eb181dc85ffa12ea67/model/utils.go#L214 to pass `team_id` to the server.

Reply:
Thanks for your feedback @ali-farooq0    I have moved that to model/command_request.go and renamed the struct to CommandMoveRequest.